assets/imgs/tab-list.svg

```diff
@@ -0,0 +1 @@
13-65.89h0c30.12,0,26.87,20.58,26.87,50.69Z" transform="translate(-144.63 -186.3)" fill="#6c63ff"/><path d="M777.61,162.68h17.08a0,0,0,0,1,0,0v23.67a7.87,7.87,0,0,1-7.87,7.87h-1.34a7.87,7.87,0,0,1-7.87-7.87V162.68A0,0,0,0,1,777.61,162.68Z" fill="#fda57d"/><polygon points="748.7 304.82 731.62 383.66 801.26 467.76 819.65 453.31 764.24 375.78 788.78 304.82 748.7 304.82" fill="#e0e0e0"/><polygon points="748.7 304.82 731.62 383.66 801.26 467.76 819.65 453.31 764.24 375.78 788.78 304.82 748.7 304.82" opacity="0.04"/><polygon points="788.12 304.82 770.47 403.68 784.86 486.15 808.97 486.15 802.4 400.75 828.85 304.82 788.12 304.82" fill="#e0e0e0"/><path d="M915.1,405.51" transform="translate(-144.63 -186.3)" fill="#6c63ff"/><path d="M961.76,443.49l-53.17,38,4.34,7.6,62.16-31.53a16.69,16.69,0,0,0,8.24-20.32C975.92,415.74,961.76,443.49,961.76,443.49Z" transform="translate(-144.63 -186.3)" opacity="0.05"/><path d="M947.46,399.72,965,444.57,910.76,476l5.43,14.11,62.16-31.53a16.69,16.69,0,0,0,8.24-20.32c-7.41-21.47-22.72-59.41-35.71-50.49A10,10,0,0,0,947.46,399.72Z" transform="translate(-144.63 -186.3)" opacity="0.05"/><path d="M946.38,397.55l17.55,44.86-53.17,35.81L914,488l63.24-31.53a16.69,16.69,0,0,0,8.24-20.32c-7.41-21.47-22.72-59.41-35.71-50.49A10,10,0,0,0,946.38,397.55Z" transform="translate(-144.63 -186.3)" fill="#6c63ff"/><path d="M953.08,672.46h-25l-32.79,6.15a2.37,2.37,0,0,0-1.93,2.33h0a2.37,2.37,0,0,0,2.37,2.37h57.31Z" transform="translate(-144.63 -186.3)" fill="#333"/><path d="M948.74,400.08s13,17.36,10.85,30.38" transform="translate(-144.63 -186.3)" opacity="0.05"/><path d="M922.24,371a29.29,29.29,0,0,0,17.08.92V351.15H922.24Z" transform="translate(-144.63 -186.3)" opacity="0.05"/><circle cx="787.66" cy="155.08" r="29.2" fill="#fda57d"/><path d="M800.07,412s-28.21,4.34-27.13,8.68,27.13,1.09,27.13,1.09Z" transform="translate(-144.63 -186.3)" fill="#fda57d"/><path d="M911.48,478.42s-25.25,13.32-22.8,17.07,26-7.84,26-7.84Z" transform="translate(-144.63 -186.3)" fill="#fda57d"/><path d="M910.76,405.51s-21.87,4.78-5.51,30.38" transform="translate(-144.63 -186.3)" fill="#6c63ff"/><path d="M936.8,407.68s-21.87,4.78-5.51,30.38" transform="translate(-144.63 -186.3)" opacity="0.05"/><path d="M937.89,407.68s-21.87,4.78-5.51,30.38" transform="translate(-144.63 -186.3)" fill="#6c63ff"/><path d="M954,350.75c-5.77,9.35-5.42,21.49-1.6,31.79s10.73,19.12,17.73,27.59c6.56,7.94,13.59,15.95,22.91,20.33,18.85,8.86,41.44.42,58.35-11.73-2.68,1.14-5.49,2.31-8.39,2.12s-5.92-2.2-6.22-5.09c-.35-3.46,3-6,5-8.83,3.62-5,3.18-12.28-.08-17.55s-8.92-8.72-14.87-10.5c-4.93-1.47-10.73-2.29-13.57-6.58-4.6-7,2.52-15.74,3.13-24.08.59-7.94-6-15.73-13.92-16.49-4.59-.44-10,1-13.3-2.24a9,9,0,0,1-2-3.31c-2.14-5.55-1.14-11.74-.87-17.68s-.55-12.62-5.12-16.42c-2.65-2.2-6.14-3.07-9.54-3.62-10.3-1.68-20.85-1.11-31.17.41-14.39,2.11-29.37,6.58-38.9,17.56s-10.77,30,.73,38.94c-4.07-3.15,15.77-21.7,25.09-16C938,335.81,938.22,351.37,954,350.75Z" transform="translate(-144.63 -186.3)" fill="#333"/></svg>
```

Comment:
Rockin' the SVG - love it.  Nice incorporation of a new topic not discussed during UI.